src/PerformanceTests/Transport.V7.AmazonSQS.V4/AmazonSQSProfile.cs

```diff
@@ -20,14 +22,26 @@ public void Configure(EndpointConfiguration endpointConfiguration)
         // https://docs.particular.net/transports/sqs/configuration-options
         var transport = endpointConfiguration.UseTransport<SqsTransport>();
 
+        NServiceBus.Logging.LogManager.GetLogger(nameof(AmazonSQSProfile)).Info(RegionEndpoint.GetBySystemName(cs["Region"].ToString()).DisplayName);
```

Comment:
Formatting

Reply:
This was actually just for testing, I'll remove it.